src/content/api/accounts.mdx

```diff
@@ -59,6 +59,10 @@ individual account.
   <Property name="subscriptions" type="array">
     A list of [Subscriptions](#subscription-model) on the Account.
   </Property>
+
+  <Property name="parentAccountId" type="string">
+    If the account was created by another Centrapay account and that account has permission to create child accounts, this will be the id of the parent's account.
```

Comment:
I think we should focus the description to be more about what the relationship between the parent and child. I think we're going into too much low level detail here.

Reply:
Suggestion: 
> The unique identifier of the parent account to which the current account is linked. This field establishes a hierarchical relationship between accounts, indicating that the current account was created by the specified parent account.